test/execution/test_plan.py

```diff
@@ -40,6 +40,21 @@ def plan_abs(data):
     assert plan_abs.is_built
 
 
+def test_tracing_torch():
+    @sy.func2plan()
+    def plan_torch(x, torch=th):
+        a = torch.rand([2])
+        x = torch.mul(a, x)
+        return torch.split(x, 2)
+
+    plan_torch.build(th.tensor([1, 2]))
+    plan_torch.forward = None
```

Comment:
Q: This is needed such that the local worker will run the plan using the actions? (not calling the function directly)

Reply:
Yes, just a pseudo hack to really use the actions traced and not the forward function